node/risk-service/server/routes/__tests__/risk.js

```diff
@@ -0,0 +1,302 @@
+    db._queryResult([
+      {
+        calculate_flood_risk: {
+          in_england: true,
+          flood_alert_area: [
+            '033WAF204'
+          ],
+          flood_warning_area: [
+            '033FWF3AVON013'
+          ],
+          rofrs_risk: {
+            prob_4band: 'Medium',
+            suitability: 'Street to Parcels of land',
+            risk_for_insurance_sop: 'Yes'
+          },
+          surface_water_risk: 'High',
+          reservoir_risk: [
+            {
+              resname: 'Draycote Water',
+              risk_desig: 'High Risk',
+              location: '445110, 270060',
+              ut_company: 'Severn Trent Water Authority',
+              ea_area: 'Environment Agency - Staffordshire, Warwickshire and West Midlands',
+              llfa_name: 'Warwickshire',
+              comments: 'If you have questions about local emergency plans for this reservoir you should contact the named Local Authority'
+            }
+          ],
+          surface_water_suitability: 'County to Town',
+          lead_local_flood_authority: 'Warwickshire',
+          extra_info: null
+        }
+      }
+    ])
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toEqual(STATUS_CODES.HTTP_STATUS_OK)
+  })
+
+  test('/floodrisk/{x}/{y}/{radius} - Invalid db result', async () => {
+    db._queryResult([{ error_result: '' }])
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toEqual(STATUS_CODES.HTTP_STATUS_BAD_REQUEST)
+  })
+
+  test('/floodrisk/{x}/{y}/{radius} - Groundwater alert result', async () => {
+    db._queryResult([
+      {
+        calculate_flood_risk: {
+          in_england: true,
+          flood_alert_area: [
+            '033WAG204'
+          ],
+          flood_warning_area: [
+            '033FWF3AVON013'
+          ],
+          rofrs_risk: {
+            prob_4band: 'Medium',
+            suitability: 'Street to Parcels of land',
+            risk_for_insurance_sop: 'Yes'
+          },
+          surface_water_risk: 'High',
+          reservoir_risk: [
+            {
+              resname: 'Draycote Water',
+              risk_desig: 'High Risk',
+              location: '445110, 270060',
+              ut_company: 'Severn Trent Water Authority',
+              ea_area: 'Environment Agency - Staffordshire, Warwickshire and West Midlands',
+              llfa_name: 'Warwickshire',
+              comments: 'If you have questions about local emergency plans for this reservoir you should contact the named Local Authority'
+            }
+          ],
+          surface_water_suitability: 'County to Town',
+          lead_local_flood_authority: 'Warwickshire',
+          extra_info: null
+        }
+      }
+    ])
+
+    const response = await server.inject(options)
+    expect(response.statusCode).toEqual(STATUS_CODES.HTTP_STATUS_OK)
+  })
+
+  test('/floodrisk/{x}/{y}/{radius} - Groundwater warning result', async () => {
```

Comment:
These tests can probably be refactored so the db._queryResults argument can be a single object with default values and arguments for any changes. Should reduce the file size and make it more readable.